lib/min_heap.rb

```diff
@@ -7,25 +7,33 @@ def initialize(key, value)
   end
 end
 
-class MinHeap
+class MinHeap # every node is smaller than or equal to it's children
 
   def initialize
     @store = []
   end
 
   # This method adds a HeapNode instance to the heap
-  # Time Complexity: ?
-  # Space Complexity: ?
+  # Time Complexity: O(log n)
+  # Space Complexity: O(n)
   def add(key, value = key)
```

Comment:
👍 , However your space complexity due to the tree structure and recursion is O(log n)